client/src/index.css

```diff
@@ -155,7 +155,7 @@ video {
   justify-content: flex-end;
   gap: 0.4rem;
   border-top: 0.1rem solid var(--border);
-  padding: 0.2rem 0.4rem;
+  padding: 0.6rem 0.4rem;
```

Comment:
![image](https://github.com/rohanshiva/gitgame/assets/20916697/5ac3966d-fae5-4c2c-b61e-b466e166b219)

Gives a bit more padding on top of the buttons on the context menu. 

Before:
![image](https://github.com/rohanshiva/gitgame/assets/20916697/6ec9dbea-5107-4ef3-a7f0-e1c9fce7722e)